aiplatform/src/main/java/aiplatform/CreatePipelineJobModelTuningSample.java

```diff
@@ -116,3 +117,4 @@ static Value numberToValue(int n) {
 }
 
 // [END aiplatform_sdk_tuning]
+// [END generativeaionvertexai_sdk_tuning]
```

Comment:
IIUC, the region tags follow LIFO order. The last START region tag should be the first one to END?
Is this assumption right?

Reply:
Hi @Sita04  Thank you. `LIFO order` is not necessary for region tags. https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/cdn/snippets.py is a good example. (SnippetBot also accepts this order)

In an upcoming request noted at [b/355457593](http://b/355457593), we will be clearing these unwanted region tags with prefix `aiplatform_`. So I request your understanding here.

Reply:
Marking this as resolved.